pkg/arvo/app/dbug.hoon

```diff
@@ -773,7 +773,7 @@
           'next'^(numb next)
         ::
           :-  'unsent-messages'  ::  as byte sizes
-          (set-array unsent-messages (cork (cury met 3) numb))
+          (set-array unsent-messages (cork jam (cork (cury met 3) numb)))
```

Comment:
I'm worried that this could be unacceptably slow. We could move `+jim` so that it can be used here, otherwise it's probably best to remove this field or stub it out.